cli/compat/esm_resolver.rs

```diff
@@ -121,9 +129,23 @@ fn node_resolve(
   let conditions = DEFAULT_CONDITIONS;
   let url = module_resolve(specifier, &parent_url, conditions)?;
 
+  let resolve_response = if url.as_str().starts_with("http") {
```

Comment:
Nit: check `url.scheme()` instead?

Reply:
@bartlomieju do you want to address or should I?